Lib/test/_test_multiprocessing.py

```diff
@@ -649,6 +649,21 @@ def _test_close(cls, rc=0, q=None):
             q.get()
         sys.exit(rc)
 
+    def test_is_close(self):
```

Comment:
You could add the test checking that the queue is closed directly in the test that closes the queue.

Reply:
I don't think it's a problem to add it here. This class is used to test the Process method. And there is a corresponding close test below.

Reply:
Yes, but I mean't that you don't really need to duplicate lines. Also, what you are essentially doing is checking that `.close()` indeeed ... closes the process.

Reply:
I seem to have misunderstood you. I said to make changes.

Reply:
I still think you can add the `self.assertFalse(p.closed)` and `self.assertTrue(p.closed)` directly in `test_close` instead of adding yet-another test which only duplicates `test_close`.